src/cdk-experimental/menu/menu-item.ts

```diff
@@ -81,6 +83,11 @@ export class CdkMenuItem implements FocusableOption {
     this._elementRef.nativeElement.focus();
   }
 
+  // In Ivy the `host` metadata will be merged, whereas in ViewEngine it is overridden. In order
+  // to avoid double event listeners, we need to use `HostListener`. Once Ivy is the default, we
+  // can move this back into `host`.
```

Comment:
Include a formal 'TODO' to do this?

Reply:
This is the standard comment we use throughout the library; we have a backlog item to clean them all up when we can get to deleting ViewEngine